packages/plugin-core/src/commands/PublishPod.ts

```diff
@@ -90,7 +90,12 @@ export class PublishPodCommand extends BaseCommand<
     }
   }
 
-  async showResponse() {
-    window.showInformationMessage("contents copied to clipboard");
+  async showResponse(resp: any) {
```

Comment:
Where does this get invoked? Is it possible to move the github specific behavior into an override method within `GithubPod`?

Reply:
It gets invoked from [here](https://github.com/dendronhq/dendron/blob/master/packages/plugin-core/src/commands/base.ts#L85).

Reply:
Ok for to leave it for now since it's such a small functionality, but in general we should find ways to avoid putting child class logic into base class logic, even if it means refactoring how the existing code is laid out.